pkgs/development/compilers/elm/packages/node/elm-pages/default.nix

```diff
@@ -1,9 +1,29 @@
 { nodePkgs, pkgs, lib, makeWrapper }:
 
-nodePkgs."elm-pages".overrideAttrs (
+let
+  ESBUILD_BINARY_PATH = lib.getExe (
+      pkgs.esbuild.override {
+        buildGoModule = args: pkgs.buildGoModule (args // rec {
+          version = "0.21.5";
+          src = pkgs.fetchFromGitHub {
+            owner = "evanw";
+            repo = "esbuild";
+            rev = "v${version}";
+            hash = "sha256-FpvXWIlt67G8w3pBKZo/mcp57LunxDmRUaCU/Ne89B8=";
+          };
+          vendorHash = "sha256-+BfxCyg0KkDQpHt/wycy/8CTG6YBA/VJvJFhhzUnSiQ=";
+        });
+      }
+    );
+in nodePkgs."elm-pages".overrideAttrs (
   old: {
+    inherit ESBUILD_BINARY_PATH;
     nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ makeWrapper old.nodejs.pkgs.node-gyp-build ];
 
+    preRebuild = ''
+      sed -i 's/"esbuild": "0\.19\.12"/"esbuild": "0.21.5"/' package.json
```

Comment:
This should use substituteInPlace with --replace-fail

Reply:
Done.